### DISCUSSION TITLE
data-prefix-icon & data-suffix-icon on all outer divs

### DISCUSSION THREAD

**Discussion by lasseitorp** [formkit/formkit#1575] [Ideas]
Currently only certain inputs get the `data-prefix-icon` and the `data-suffix-icon` prop when an icon is added. As an example the `select` gets it, but the `dropdown` doesn't.

For consistency it would be nice if all related inputs get the same data props.

**Comment by sashamilenkovic** [comment]
@lasseitorp This has been resolved https://github.com/formkit/formkit/issues/1577
